extension/src/openvic-extension/utility/UITools.cpp

```diff
@@ -123,6 +128,69 @@ static Array get_events_from_shortcut_key(String const& key) {
 	return events;
 }
 
+static Error try_create_shortcut_action_for_button(
+	GUIButton* gui_button, String const& shortcut_key_name, String const& shortcut_hotkey_name = ""
+) {
+	Array event_array = get_events_from_shortcut_key(shortcut_key_name);
+
+	ERR_FAIL_COND_V_MSG(
+		!shortcut_key_name.is_empty() && event_array.is_empty(), ERR_INVALID_PARAMETER,
+		vformat("Unknown shortcut key '%s' for GUI button %s", shortcut_key_name, gui_button->get_name())
+	);
+
+	if (shortcut_key_name.is_empty()) {
+		return OK;
+	}
+
+	InputMap* const im = InputMap::get_singleton();
+	String action_name;
+	if (shortcut_hotkey_name.is_empty()) {
+		action_name = //
+			vformat("button_%s_hotkey", gui_button->get_name().to_lower().replace("button", "").replace("hotkey", ""))
+				.replace("__", "_");
+	} else {
+		action_name = vformat("button_%s_hotkey", shortcut_hotkey_name);
+	}
+	Ref<InputEventAction> action_event;
+	action_event.instantiate();
+	action_event->set_action(action_name);
+	action_event->set_pressed(true);
+
+	if (im->has_action(action_name)) {
+		TypedArray<InputEvent> events = im->action_get_events(action_name);
+		bool skip_warning = events.size() != event_array.size();
+		if (!skip_warning) {
+			for (std::size_t index = 0; index < events.size(); index++) {
+				if (!event_array.has(events[index])) {
+					skip_warning = false;
+					break;
+				}
+			}
+		}
```

Comment:
This block only triggers if `skip_warning` is false, and all it can do is `skip_warning = false`, so either it's unnecessary or there's a mistake?